.vscode/tasks.json

```diff
@@ -0,0 +1,23 @@
+{
+    "tasks": [
+        {
+            "type": "cppbuild",
+            "label": "build cloe",
+            "command": "make",
+            "args": [
+                "all",
+                "CONAN_OPTIONS=\"-c tools.build:skip_test=1\"",
+                "CONAN_RUN_TESTS=0"
```

Comment:
You could add `BUILD_TYPE=Debug` if you want to force debug build type.

Alternatively, the user can set this kind of thing up themselves.